paddle/fluid/pybind/eager_method.cc

```diff
@@ -1444,6 +1445,35 @@ static PyObject* tensor__copy_gradient_from(TensorObject* self,
   EAGER_CATCH_AND_THROW_RETURN_NULL
 }
 
+static PyObject* tensor__set_use_cudnn(TensorObject* self,
```

Comment:
这里既然是set方法，为何还要返回一个tensor对象，感觉命名有点不规范

Reply:
修改了函数的命名。此函数需要返回 tensor 对象，返回的 tensor 与原 tensor 相比，只有 use_cudnn 属性不一致。
一个 tensor 可以同时作为多个 op 的输入，如果对 tensor 使用 inplace 的 set 方法，会污染其他 op 的行为，因此需要返回一个新的 tensor。
例如 Tensor x 同时作为 conv2d 和 depthwise_conv2d 的输入，如果只有 depthwise_conv2d 想要使用 plain GPU kernel，直接 set x.use_cudnn = False 时，conv2d 也会选取 plain GPU kernel。因此需要返回一个新的 tensor 隔离 op 之间选择 kernel 的方式。

Thx~, I have changed the name of this function. This function has to return a tensor object, the returned tensor is the same as the original tensor except `use_cudnn` attribute.
A tensor can be the input of multiple ops at the same time. It will contaminate other ops' action if we set `use_cudnn` attribute inplace, thus we need to return a new tensor.
For example, Tensor `x` is the input of both `conv2d` and `depthwise_conv2d`. However, only `depthwise_conv2d` wants to use plain GPU kernel and `conv2d` intends to use CuDNN kernel. If we set `x.use_cudnn=False` directly, `conv2d` will select plain GPU kernel incorrectly. Hence we need to return a new tensor to isolate the action between multiple ops.